### DISCUSSION TITLE
MemoryError issus when navigating the app

### DISCUSSION THREAD

**Discussion by Virginie74** [whitphx/stlite#1127] [Q&A]
Hi,

Thanks a lot for the amazing job your doing with stlite. It corresponds exactly to my need and the library is very well documented making easy to transform a streamlit app into an executable file.

I manage to get the dist folder containing the executable file. Once the latter open, I can navigate my app but shorthly I get a pink box with a MemoryError message finishing with File "<frozen codecs>", line 322, in decode,. 

Do you know why and how can I avoid this?

All the best,
Virginie

**Comment by whitphx** [comment]
Hi, can you provide more details, e.g. the full code to reproduce the error and the full error log?
